website/blog/2021-08-18-improving-marker-mechanism.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Improving Marker Mechanism in Apache Hudi"
+excerpt: "We introduce a new marker mechanism leveraging the timeline server to address performance bottlenecks due to rate-limiting on cloud storage like AWS S3."
+author: yihua
+category: blog
+---
+Write operations in an Apache Hudi table use markers to efficiently identify the data files written to the file system.  In this blog, we dive into the design of the existing direct marker file mechanism and explain its performance problem on cloud storage like AWS S3.  We demonstrate how we improve the write performance with timeline-server-based markers.
+
```

Comment:
@yihua - Can you please add a `<!--truncate-->` tag here, so that the main blog page will render only up to this tag with a read more link?

When we don't have the truncate tag, the full blog post will be rendered on the blog homepage.

Reply:
Got it.  Fixed.